.deliver/config

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Be nice to have a comment referring to https://github.com/gerhard/deliver here, otherwise it's a mystery